MIERUNE/qgis-plugx-plugin
# Fix plugin installation

## Issue

related #160 

## 変更内容:Description

- Fix install error due to conflict with another plugin
- Makes utils.py independant ./utils.py -> ./plugx_utils.py 

## テスト手順:Test

- Reproduce install error : 
  - Disable PlugX plugin
  - Install qgis2web plugin
  - Relaunch QGIS
  - Enable PlugX message
  - Should have this error message
 
<img width="830" alt="image" src="https://github.com/MIERUNE/qgis-plugx-plugin/assets/26103833/65671473-9d49-476c-a758-90103402982c">


- Test fixed version : 
  - Close QGIS
  - Switch to `fix/install-error` branch
  - Relaunch QGIS
  - Install above plugin
  - Check if plugin works
